src/Lumper.UI/ViewModels/Pages/Jobs/JobMenuItem.cs

```diff
@@ -5,15 +5,13 @@ namespace Lumper.UI.ViewModels.Pages.Jobs;
 
 public interface IJobMenuItem<out T>; // Interface for covariance
 
-public class JobMenuItem<T>(JobsViewModel jobsVm) : IJobMenuItem<T> where T : IJob
+public class JobMenuItem<T>(JobsViewModel jobsVm, Func<T> factory) : IJobMenuItem<T> where T : IJob
```

Comment:
there was a commit earlier where  this was done for a lot of things right? can we squash or is this different somehow? (talking about the whole commit not this line)

Reply:
I can fixup this down. This was just in a separate commit this it was a refactor I had to do after when getting release builds working, after finishing the major code changes in this PR.